images/php-fpm/Dockerfile

```diff
@@ -14,6 +14,10 @@ RUN set -xe; \
         # allow mailing to work
         sendmail \
     # pecl installs
+    && apt-get -yqq install exiftool \
```

Comment:
This can go up with the `apt-get install` stuff, right above it. You just need to move the package to the list of packages and add a comment for why its needed, like the others.